src/test/java/SetTest.java

```diff
@@ -0,0 +1,45 @@
+import java.util.HashSet;
+import java.util.Set;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    // Test Case 구현
+
+    @Test
+    void testSetSizeIs3() {
+        Assertions.assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함
static import도 사용해 보면 어떨까?

Reply:
@javajigi 
안녕하세요 선생님
궁금한것이 있습니다.
`static import`는 코드를 간결하게 만들어주는 장점이 있지만, 과도하게 사용하면 코드의 출처를 파악하기 힘들어 가독성을 떨어뜨릴 수 있다고 판단되는데요.
`Assertions.assertThat`은 일반적으로 많이들 알고 쓰이는 메서드라 static import를 권유해주신걸까요?

Reply:
static import의 과도한 사용은 오히려 가독성을 떨어트릴 수 있는 점은 공감해요.
하지만 assertThat의 경우 워낙 자주 사용하는 메서드이다보니 Assertions.assertThat이라는 점을 명확히 알 수 있기 때문에 테스크 코드의 간결함을 위해 굳이 사용하지 않을 이유는 없다고 생각해요.

Reply:
@javajigi 
네 공감합니다.
답변 감사합니다.